app/api/claim/route.ts

```diff
@@ -6,14 +6,23 @@ import { Chain } from "@/types/common";
 import { parseAvailAmount } from "@/utils/parseAmount";
 import { substrateConfig } from "@/config/walletConfig";
 import { SignerOptions } from "@polkadot/api/types";
+import { Ratelimit } from '@upstash/ratelimit';
+import { kv } from '@vercel/kv';
```

Comment:
How this kv feat will impact on billing?

Reply:
https://vercel.com/docs/storage/vercel-kv/usage-and-pricing#pricing

shouldn't be much afaik considering this is internal, but you can have a look.
i was also wondering if we can add this to the other apis(ex getTokenPrice)